README.md

```diff
@@ -7,8 +7,8 @@
 In case you need any help installing any of this, please ask your study buddy or come to one of the b-it-bots labs (C025 and C069).
 
 ### Software
-*   [Ubuntu 16.04](https://www.ubuntu.com/download/desktop)
-*   [ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu)
+*   [Ubuntu 16.04](http://releases.ubuntu.com/16.04.6/)
+*   [ROS Kinetic](http://wiki.ros.org/kinetic/Installation/Ubuntu). Installation instructions can be found [here](https://github.com/hasauino/foundations_course/tree/master/content/ros/ROS%20Installation)
```

Comment:
Please change this link so that it points to this repository and not yours.

Reply:
done